crates/nargo/tests/test_data/struct_inputs/Prover.toml

```diff
@@ -1,5 +1,4 @@
 x = "5"
-return = ""
```

Comment:
you should not change the behaviour of the return

Reply:
The return value is not needed anymore since we are no longer applying constraints.